packages/core/src/lib/light_push/utils.ts

```diff
@@ -21,3 +23,39 @@ export const isRLNResponseError = (info?: string): boolean => {
     info.includes(RLN_REMOTE_VALIDATION)
   );
 };
+
+export function isSuccess(statusCode: number | undefined): boolean {
+  return statusCode === LightPushStatusCode.SUCCESS;
+}
+
+export function toLightPushError(
+  statusCode: LightPushStatusCode | number | undefined
+): LightPushError {
+  if (!statusCode) {
+    return LightPushError.GENERIC_FAIL;
+  }
+
+  switch (statusCode) {
```

Comment:
this mapping looks terrible, almost all branches are `LightPushStatusCode.INVALID_MESSAGE` -> `LightPushError.INVALID_MESSAGE`

why?